frontend/src/components/Loader.vue

```diff
@@ -1,63 +1,68 @@
 <script setup lang="ts">
-import { generateRange } from '../utilities';
+import { generateRange } from '../utilities/utils';
```

Comment:
When we already have a subfolder named `utilities`, the files/modules inside of it should ideally have clear reponsibilities. I think that here `utils` could be renamed e.g. to `api` or `requests`, and the storage stuff could be moved to a different file called e.g. `storage`. `generateRange` can just be inside `Loader.vue`.